Client/YooChooseNotifier.php

```diff
@@ -124,13 +104,27 @@ public function setIncludedContentTypes($value)
         $this->options['included-content-types'] = $value;
     }
 
+    /**
+     * Checks if notifier has configuration.
+     *
+     * @return bool
+     */
+    public function hasCredentials()
+    {
+        return !empty($this->options['customer-id']) && !empty($this->options['license-key']);
+    }
+
     /**
      * {@inheritdoc}
      */
     public function updateContent($contentId, $versionNo = null)
     {
         try {
-            $content = $this->contentService->loadContent($contentId, null, $versionNo);
+            if (!$this->hasCredentials()) {
+                return;
```

Comment:
Rather idea than change request: what do you think about writing to the logs that YooChoose couldn't be notified due to the lack of credentials?

Reply:
I think we should rather find a better way for conditional registration of Slots `ezpublish.api.slot` in this bundle, this is just the simple solution for https://jira.ez.no/browse/DEMO-149